stl/inc/format

```diff
@@ -663,8 +664,10 @@ public:
     }
 
     constexpr void _On_zero() {
-        _Specs._Alignment = _Align::_None;
-        _Specs._Fill[0]   = _CharT{'0'};
+        _Specs._Leading_zero = true;
+        if (_Specs._Alignment == _Align::_None) {
+            _Specs._Fill[0] = _CharT{'0'};

```

Comment:
Actually we should probably set the other fill characters to `0` so that we don't get something gross.

Reply:
after some discussion we don't actually think this matters, multibyte fills only come from user-specified fills.